fdbserver/DataDistribution.actor.cpp

```diff
@@ -3175,8 +3175,21 @@ ACTOR Future<Void> serverMetricsPolling( TCServerInfo *server) {
 	}
 }
 
-//Returns the KeyValueStoreType of server if it is different from self->storeType
-ACTOR Future<KeyValueStoreType> keyValueStoreTypeTracker(DDTeamCollection* self, TCServerInfo *server) {
+//Returns if the KeyValueStoreType of server is different from self->storeType or the desired datacenter does not match
```

Comment:
How does this solve the problem mentioned in the commit message?
If a SS is offline, shouldn't it be detected as failed and removed in the `storageServerTracker`?